app/models/admin_setting.rb

```diff
@@ -50,6 +50,9 @@ def self.tag_wrangling_off?
   def self.guest_downloading_off?
     self.first ? self.first.guest_downloading_off? : false
   end
+  def self.downloads_enabled?
```

Comment:
This pull request is already approved, but if you have a change to add a blank line before and after this `def self.downloads_enabled?` method, that would be great.